### ISSUE TITLE
about qat_engine_init

### ISSUE THREAD

**Issue by konishi-av** [01org/QAT_Engine#4]
Hello steve, excuse me, but

https://github.com/01org/QAT_Engine/issues/1#issuecomment-228795331
@stevelinsell wrote on 28 Jun:

> Yes, I have also encountered this issue a couple of weeks ago. The development team are addressing it during this development cycle which has just started so if all goes to plan you should expect a fix to be posted to here within the next two weeks.

When is the issue mentioned above fixed?
Please let me know the schedule, if possible.

Kind regards,

Tom.

**Comment by stevelinsell** [comment]
Hi Tom,

Firstly apologies for the slowness of our updates. Our intended release process is to collate commits and then make a release every 2 weeks. We ran into some problems that delayed our last two releases. The next release is scheduled for the end of this week (late today or tomorrow). We are running through our release process now and are not anticipating any issues this time.

As a heads up the biggest change you'll see in the coming release will be that the engine has been changed to use an autotools generated install. It now comes with a standard configure script rather than having to set numerous environment variables. We would be interested in any feedback on this change.

Kind Regards,

Steve.
